Makefile

```diff
@@ -9,23 +14,24 @@ else
     DETECTED_OS := $(shell uname)
 endif
 
+.PHONY: binaries/client
```

Comment:
Why do we want to download client binaries on every run?

Reply:
I think I added this because it doesn't doesn't check correctly the folder modification date. Plus the downloaders already check and verify the current versions before downloading again.